### DISCUSSION TITLE
<title> two factor authentication problem</title>

### DISCUSSION THREAD

**Discussion by Kidusan-Teklu** [community/community#121253] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

when  i try to  enable two factor authentication by my phone there is no option on the country code list for Ethiopia (+251)

**Comment by fdistasio** [comment]
Ethiopia is not supported for SMS authentication, alternatively you can use the two factor authentication via an authenticator app.
List of supported Countries: https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/countries-where-sms-authentication-is-supported

**Reply by fdistasio** [reply]
If you received the response you wanted, please close this discussion marking an answer.

**Comment by Kidusan-Teklu** [comment]
try your best before  forcing  people from countries where sms is not supported
